mobile/native/src/api_lndlc/lndlc.rs

```diff
@@ -47,6 +51,18 @@ pub fn run(stream: StreamSink<Event>, data_dir: String) -> Result<()> {
 
         let background_processor = node.start().await?;
 
+        // todo: should the library really be responsible for managing the task?
```

Comment:
❓What are you referring to here? The fact that we use `keep_connected` and don't manage this here? Or the fact that we manage the connection here and not in Dart? 

My perspective on Rust vs. Dart:
Stateful presentation components such as Change notifiers to track value changes are OK; but managing the conneciton is on an application state level, and I would have seen that in the responsibility of the rust code.